custom-targets/vertex-ai/quickstart/QUICKSTART.md

```diff
@@ -98,38 +109,29 @@ take 5 minutes or so.
 
 The endpoint ID will be used to refer to the endpoint, rather than the display name.
 
-## 6. Build and Register a Custom Target Type for Vertex AI
-
-From within the `quickstart` directory, run this command to build the Vertex AI model deployer image and
-install the custom target resources:
-
-```shell
-../build_and_register.sh -p $PROJECT_ID -r $REGION
-```
-
-For information about the `build_and_register.sh` script, see the [README](../README.md#build)
-
-## 7. Create delivery pipeline, target, and skaffold
+## 6. Create delivery pipeline, target, and skaffold
```

Comment:
Capitalize Skaffold, and I think you should say "...and Skaffold config" rather than just "...and Skaffold."